packages/nitro-protocol/contracts/AssetHolder.sol

```diff
@@ -422,8 +411,26 @@ contract AssetHolder is IAssetHolder {
                 )
             );
         } else {
+            delete assetOutcomeHashes[guarantorChannelId];
```

Comment:
If there's nothing stored for the target, we may as well delete the guarantor and get a gas rebate.